### ISSUE TITLE
Infinite stone mask can and does spawn in spessmart

### ISSUE THREAD

**Issue by Kammerjunk** [d3athrow/vgstation13#14234]
#### Description of issue
The Stone Mask Infinite Edition spawns in spessmart. It costs 150 credits and turns anyone (and everyone - it's infinite) into a vampire. Is this really a good idea?
![image](https://cloud.githubusercontent.com/assets/5942183/23435473/e0d0d800-fe07-11e6-8f19-6cccc3364348.png)

#### Difference between expected and actual behavior
Instead of a fun trip to the mall you get a 20-man vampfest for the small price of some blood.

#### Steps to reproduce 
1. Load up spessmart
2. Repeat step one if the mask did not spawn
3. Check out that the mask is infinite
4. Turn everyone into a vamp

#### Specific information for locating

I'm guessing it happens because of this:
https://github.com/d3athrow/vgstation13/blob/c2163cfd52ae751de38055d8747d23e235c30846/maps/randomvaults/spessmart.dm
`var/list/clothing = existing_typesof(/obj/item/clothing) - typesof(/obj/item/clothing/suit/space/ert) - typesof(/obj/item/clothing/head/helmet/space/ert) - list(/obj/item/clothing/suit/space/rig/elite, /obj/item/clothing/suit/space/rig/deathsquad, /obj/item/clothing/suit/space/rig/wizard, /obj/item/clothing/head/helmet/space/bomberman, /obj/item/clothing/suit/space/bomberman) //What in the world could go wrong`

#### Length of time in which bug has been known to occur

Since spessmart was introduced, presumably

#### Server revision

f72fecb701a91a944378edf4eb41dfdbe4a3934b

#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of not being yelled at by every contributor.

- [x] Issue could be reproduced at least once
- [x] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues) Searched "stone mask" and "spessmart"

**Comment by Fruktlimpa** [comment]
Make it one use only and the problem is solved. Unless getting a F R E E antag ticket with a little time(and luck) spent in spess is too much.

**Comment by Kammerjunk** [comment]
Hardly too much. The syndie vault allows for the same thing (after killing a bunch of simple animal syndies and there's a 50% chance it'll blow you up or whatever it does)

**Comment by Yelier** [comment]
Spessmart is fucking crazy. The goonecode spawns there for only $400 (or free if you're a diona). That's one wish for no effort at all, which is pretty damn powerful if you ask me.

**Comment by Probe1** [comment]
We basically never fulfill goonecode wishes anymore since no matter where you get it, you probably cheesed it.

**Comment by Yelier** [comment]
I'm speaking from experience, just last night I found a spessmart with a goonecode and wished to go to Centcomm. I could have grabbed an admin ID, staff of destruction, vampire mask, and a seraph and gone back to the station via ERT/deathsquad shuttle and fucked up everything, but I'm not a massive dick, so instead I partied with some other guys there. Unless you give an admin briefing to just not fulfil goonecode wishes, it really just depends on the admin.
